coding.md

```diff
@@ -217,4 +217,8 @@ It is not advised to use preprocessor conditionals like `#if` or `ifdef'. The us
 Following notation for operating system messages should be applied. Message should start from a subsystem name, which should be followed by colon and a message body. An example is shown below.
 
 >
-    lib_printf("main: Starting syspage programs (%d) and init\n", syspage->progssz);
\ No newline at end of file
+    lib_printf("main: Starting syspage programs (%d) and init\n", syspage->progssz);
+
+##Miscellaneous
```

Comment:
I've looked at http://phoenix-rtos.com/documentation/coding.md instead of github's page and there it seems like headers are ok. 
However, you have right, headers are not rendered correctly. I have made changes to the whole file. 
Thanks for notice!